### PR TITLE
fix: Prevent deleted users from breaking user mapping on FineView

### LINKED ISSUE #212: [BUG] Clicking Apply Dates on Fine Overview Gives an Error
### What browsers are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### Is this happening on the SudoSOS point of sale or on the SudoSOS dashboard?

Dashboard

### Current behaviour

When clicking apply on the fine overview page, nothing happens and an error is given in the console about gewisId being undefined.

### Desired behaviour

When clicking apply, a list of users should be given who are eligible for a fine based on the reference dates.

### Steps to reproduce

1. Login on the dashboard using an account with permission to hand out fines.
2. Go to the fine overview
3. Select two dates
4. Click apply
5. Open console

### Other information

_No response_

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
# Description
Fixes issue where deleted and inactive users were considered in a mapping for the FineView and breaking the mapping when trying to find this user.

## Related issues/external references
Fixes #212.

## Types of changes
- Bug fix _(non-breaking change which fixes an issue)_

### CODE REVIEW

File: apps/dashboard/src/main.ts
Comment (SuperVK) [inline apps/dashboard/src/main.ts:1]: Why not just roll back this file, nothing changed, also not relevant to the bug your fixing right?

File: apps/dashboard/vite.config.ts [hunk 1]
```diff
@@ -16,6 +16,11 @@ export default defineConfig({
   optimizeDeps: {
     exclude: ['ToastService'],
   },
+  // build: {
+  //   rollupOptions: {
+  //     external: ['@sudosos/sudosos-frontend-common'],
+  //   },
+  // },
```
Comment (SuperVK) [inline apps/dashboard/vite.config.ts]: Why?

File: package-lock.json
Comment (SuperVK) [inline package-lock.json:1]: Why? Either put this in the .gitignore with a good reason, or not remove it (also seperate PR cause it does not seem related to the bug you are fixing)

File: apps/dashboard/vite.config.ts (hunk 1 above)
Comment (CodeNamedRobin) [reply apps/dashboard/vite.config.ts]: Fixed

File: package-lock.json
Comment (CodeNamedRobin) [reply package-lock.json:1]: Fixed.

File: apps/dashboard/src/main.ts
Comment (CodeNamedRobin) [reply apps/dashboard/src/main.ts:1]: True, rolled back.

**Review by SuperVK — APPROVED** [review]

**Pull request merged**